Casks/pdk.rb

```diff
@@ -10,12 +10,16 @@
     sha256 '2c1c39ee111be3325c8bc2e36323d3b60d504ba5038053a1aa8d4d3567f79642'
   when '10.13'
     os_ver = '10.13'
-    version '1.16.0.2'
```

Comment:
Do you also need to remove 10.11 and 10.12? Thought the e-mail said we stopped packaging those.

Reply:
Oh, I guess they're just stuck on an older version.

Reply:
I think as it stands they will just still get 1.16.0.2? Guess we can decide if we want them to just go away entirely.

Reply:
I think it's fine for this release if 10.11 and 10.12 users just get the old version still and then we can remove them entirely in a subsequent release if we want.

Reply:
The template kept them in because that's what I did for Puppet: shipped the newest version of the package that works on that OS version.